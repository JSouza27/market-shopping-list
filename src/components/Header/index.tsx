export default function Header() {
  return (
    <header className="w-full mb-6 max-w-2xl">
      <h1 className="text-2xl font-bold text-dark">Lista de Compras</h1>
    </header>
  );
}
