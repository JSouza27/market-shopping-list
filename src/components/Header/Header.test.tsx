import { render, screen } from '@testing-library/react';

import Header from '.';

describe('<Header />', () => {
  it('should render the heading', () => {
    const { container } = render(<Header />);

    expect(
      screen.getByRole('heading', { name: /Lista de Compras/i })
    ).toBeInTheDocument();

    expect(container.firstChild).toMatchSnapshot();
  });
});
