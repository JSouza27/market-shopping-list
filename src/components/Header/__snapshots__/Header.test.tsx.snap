// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Header /> should render the heading 1`] = `
<header
  class="w-full mb-6 max-w-2xl"
>
  <h1
    class="text-2xl font-bold text-dark"
  >
    Lista de Compras
  </h1>
</header>
`;
